### PR TITLE
test(login): fix login integration test

### PR DESCRIPTION
**Pull request by baurine**

### CODE REVIEW

File: pkg/apiserver/user/auth.go [hunk 1]
```diff
@@ -122,6 +122,15 @@ func NewAuthService(featureFlags *featureflag.Registry) *AuthService {
...
 			}
+			if form.Type == 0 {
+				// generate new rsa key pair for each sql auth login
+				privateKey, publicKey, err := GenerateKey()
+				// if generate successfully, replace the old key pair
+				if err == nil {
+					service.RsaPrivateKey = privateKey
+					service.RsaPublicKey = publicKey
+				}
+			}
```
Comment (shhdgit) [inline pkg/apiserver/user/auth.go:133]: Will there be issues during concurrency?

Comment (baurine) [reply pkg/apiserver/user/auth.go:133]: Yep, this is a potential issue, but for tidb-dashboard use case, it's a very low possibility. If the user meet and think it's a question, we can consider to improve it later.

Comment (baurine) [reply pkg/apiserver/user/auth.go:133]: Hold this PR and think it clearer.

**Review by shhdgit — APPROVED** [review]

File: pkg/apiserver/user/auth.go (hunk 1 above)
Comment (baurine) [reply pkg/apiserver/user/auth.go]: Revert the key logic and change this PR to `fix login integration test`.

**Pull request merged**